domain/src/main/java/com/ataulm/artcollector/datanotdomain/GalleryRepository.kt

```diff
@@ -0,0 +1,9 @@
+package com.ataulm.artcollector.datanotdomain
+
+import androidx.paging.PagingSource
+import com.ataulm.artcollector.Painting
+
+interface GalleryRepository {
+
+    fun gallery(): PagingSource<Int, Painting>
```

Comment:
PagingSource is from paging-common, the jar.

this file is in our kotlin only module.